harvest-refactor/oai-harvest.py

```diff
@@ -0,0 +1,55 @@
+import errno
+import helper
+import os
+from argparse import ArgumentParser
+from sickle import Sickle
+from sickle.iterator import OAIResponseIterator
+
+def main():
+    # CLI client options.
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-i",
+        "--institution",
+        dest="institution",
+        help="add the institution and the set names to ignore to the helper.do_not_harvest dictionary")
+    parser.add_argument("-m", "--metadata_prefix", dest="metadata_prefix",
+                        help="add the metadata prefix if other than oai_dc")
+    parser.add_argument(
+        "-s",
```

Comment:
If you use `action='store_true'` then a user only needs to provide the flag in the commandline (no need to do `-s TRUE`).